source/sass/blocks/contacts.scss

```
@@ -0,0 +1,7 @@
+.contacts__welcome {
+  font-family: "Oswald  Regular", sans-serif;
```

```suggestion
  font-family: "Oswald", sans-serif;
```
И также во всех значениях `font-family` для данного шрифта.